cachito/workers/pkg_managers/yarn.py

```diff
@@ -90,10 +105,23 @@ def _get_deps(yarn_lock, file_deps_allowlist):
     """
     deps = []
     nexus_replacements = {}
+    non_dev_deps = []
```

Comment:
Should be this `set` ?

Reply:
100%

Reply:
On second thought, this should be a `collections.deque`.

Reply:
Actually, would using a `set` as a stack/list be more efficient than using `collections.deque`?

Reply:
Isn't this result of the function?

Reply:
Changed! Please check.